prober/derp.go

```diff
@@ -136,28 +201,52 @@ func (d *derpProber) ProbeMap(ctx context.Context) error {
 	return nil
 }
 
+// probeMesh returs a probe func that sends a test packet through a pair of DERP
+// servers (or just one server, if `from` and `to` are the same). `from` and `to`
+// are expected to be names (DERPNode.Name) of two DERP servers in the same region.
 func (d *derpProber) probeMesh(from, to string) ProbeFunc {
 	return func(ctx context.Context) error {
-		d.Lock()
-		dm := d.lastDERPMap
-		fromN, ok := d.nodes[from]
-		if !ok {
-			d.Unlock()
-			return fmt.Errorf("could not find derp node %s", from)
+		fromN, toN, err := d.getNodePair(from, to)
+		if err != nil {
+			return err
 		}
-		toN, ok := d.nodes[to]
-		if !ok {
-			d.Unlock()
-			return fmt.Errorf("could not find derp node %s", to)
+
+		dm := d.lastDERPMap
+		return derpProbeNodePair(ctx, dm, fromN, toN)
+	}
+}
+
+// probeBandwidth returs a probe func that sends a payload of a given size
+// through a pair of DERP servers (or just one server, if `from` and `to` are
```

Comment:
FWIW the use of \`markup\` in Go comments is unusual, and not something we've generally done elsewhere. These read fine even without it, and while godoc will render them, I don't think it adds much. YMMV.

Reply:
I personally regularly stumble when I read comments that refer to names of variables/functions without quoting them, so have built a habit of doing this for clarity. Changed from backticks to single quotes, perhaps that will come across as less unusual?